CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,11 @@
 
 import Foundation
 
-print("Hello, World!")
-
+// STEP 2 - 내 번호와 맞추어보기!
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+let result = checkLottoResult(myLottoNumbers)
+if result.isEmpty {
+  print("아쉽지만 겹치는 번호가 없습니다.")
+} else {
+  print("축하합니다! 겹치는 번호는 \(result.map { String($0) }.joined(separator: ", ")) 입니다!")
```

Comment:
> 아직 클로저나 map 같은 문법이 익숙지 않아서 오히려 코드를 한눈에 이해하기는 어렵네요

제가 생각했을 때에는 크게 알아보기 어렵다거나 하지는 않은데,
클로저나, 함수형 프로그래밍에 익숙하지 않다면 어색할 수 있겠네요!
Set<Int>를 받아서 String을 출력하는 함수를 만들고 내부에 저 구현을 해준뒤에
함수 이름을 정해주면 알아보기 조금 더 편할거같구요,

```suggestion
  let bracketRemovedResult: String = result.map(String.init).joined(separator: ", ")
  print("축하합니다! 겹치는 번호는 \(bracketRemovedResult) 입니다!")
```
요렇게 쓰면 코드가 조금 더 짧아질 거 같긴 하네요!